### PR TITLE
#9 instructions for git installation on linux

### PR DESCRIPTION
**Pull request by aliahmadcse**
Added necessary instructions for git setup on Linux. @koushikkothagal please review the changeset.

### CODE REVIEW

File: session-02-big-picture/README.md [hunk 1]
```diff
@@ -257,6 +257,44 @@ Possibly prerecorded videos. Need offline support
...
     
 
+## On Linux
+
+Linux comes with git pre-installed, you can check the version of git by typing the following command in terminal
+
+```console
+git --version
+```
+![git version](./assets/git-linux-setup/git-version.png)
```
Comment (koushikkothagal) [inline session-02-big-picture/README.md:267]: I suggest removing the screenshot. Just add the output in a code fence block. Like so
~~~
```console
$ git --version
git version 2.37.1
```
~~~

Notice the `$ ` to indicate prompt.

File: session-02-big-picture/README.md [hunk 2]
```diff
@@ -257,6 +257,44 @@ Possibly prerecorded videos. Need offline support
...
+```console
+sudo apt-get install git
+```
+![git installing](./assets/git-linux-setup/installing-git.png)
+
+3. After the installation is complete, set up your username and email by typing the following commands in terminal
+
+```console
+
+```console
```
Comment (koushikkothagal) [inline session-02-big-picture/README.md:286]: You might have an incorrect / duplicate ``` line here

File: session-02-big-picture/README.md [hunk 3]
```diff
@@ -257,6 +257,44 @@ Possibly prerecorded videos. Need offline support
...
+git config --global user.name "yourUserName"
+
+git config --gloabl user.email "youremail@example.com"
```
Comment (koushikkothagal) [inline session-02-big-picture/README.md:289]:
```suggestion
git config --global user.email "youremail@example.com"
```

**Review by koushikkothagal — CHANGES_REQUESTED** [review]

File: session-02-big-picture/README.md [hunk 4]
```diff
@@ -257,6 +257,43 @@ Possibly prerecorded videos. Need offline support
...
+1. Open terminal and type the following command to update the package list
+
+```console
+sudo apt-get update
+```
+2. Type the following command to install git
+```console
+sudo apt-get install git
+```
+![git installing](./assets/git-linux-setup/installing-git.png)
```
Comment (koushikkothagal) [inline session-02-big-picture/README.md:280]:
```suggestion
![git installing](../assets/git-linux-setup/installing-git.png)
```

**Review by koushikkothagal — CHANGES_REQUESTED** [review]

File: session-02-big-picture/README.md (hunk 4 above)
Comment (aliahmadcse) [reply session-02-big-picture/README.md:280]: assets exist at the same level as readme, I don't think, this needs to go one step back

Comment (koushikkothagal) [reply session-02-big-picture/README.md:280]: Ah, my bad then. Looks good!

**Review by koushikkothagal — APPROVED** [review]

**Pull request merged**